src/sharedServices/athena/createQuerySql.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { createQuerySql } from './createQuerySql'
+import {
+  testDataRequest,
+  noEventIdTestDataRequest,
+  dataPathsTestDataRequest
+} from '../../utils/tests/testDataRequest'
+
+describe('create Query SQL', () => {
+  test('returns a formatted SQL query string if all conditions satisfied', () => {
+    expect(createQuerySql(dataPathsTestDataRequest)).toEqual({
+      sqlGenerated: true,
+      sql: "SELECT json_extract(restricted, '$.user.firstName') as user_firstname, json_extract(restricted, '$.user.lastName') as user_lastname FROM test_database.test_table WHERE event_id=? OR event_id=?",
```

Comment:
I think you can use the `IN` operator here, at least according to my googling

https://docs.aws.amazon.com/athena/latest/ug/select.html

```
SELECT * FROM "webdata"."impressions"
WHERE referrer IN ('example.com','example.net','example.org')
```

Reply:
Done